### DISCUSSION TITLE
Option to keep header in the centre of the slide when using `![bg right]()`

### DISCUSSION THREAD

**Discussion by jweickm** [marp-team/marp#287] [Q&A] [answered]
Is there any simple way to tag the header so that it stays in the centre of the slide even when I am using `![bg right:30%]()` or something similar? I often add some images to the right of the slide and I would like the option to keep the header position unaffected by those.

**Comment by yhatt** [comment, marked as answer]
Refer to #49.

**Reply by jweickm** [reply]
Thanks! That should work for now! :)

**Reply by jweickm** [reply]
Follow-up: Is there a way to make this only apply to the current slide? When using the code provided in [#49](https://github.com/marp-team/marp/discussions/49) it adds the text to all of my slides.

**Reply by yhatt** [reply]
Using [scoped style](https://marpit.marp.app/theme-css?id=scoped-style) like `<style scoped> section::before { content: '...'; } </style>` should work well, but I found a bug of Marpit framework in the combination of `section::before` with `<style scoped>` and `![bg]()`. Track the progress in marp-team/marpit#328.

**Reply by jweickm** [reply]
Thank you for another lightning-fast response! I will keep my eye on the issue you linked!
